### ISSUE TITLE
Bundler::GemNotFound error after deploy (hitimes)

### ISSUE THREAD

**Issue by davidcpell** [brandonhilkert/sucker_punch#95]
After deploying my app to the staging server, I try to open the homepage in my browser and get a Rack error. The message is:

`Could not find hitimes-1.2.2 in any of the sources (Bundler::GemNotFound)`

Hitimes is a dependency of celluloid, which is a dependency of sucker_punch. Everything bundled fine in development, though. Any ideas of what might be going on?

Rails 4.2
Sucker Punch 1.3.2

*EDIT*
I found an older question about Passenger in #6, which is what I'm using, but after reading through it and the section on the readme about Passenger, it doesn't look like this is related to my issue since I don't have an initializer.

**Comment by brandonhilkert** [comment]
I don't have an issues and don't have any similar reports. Are you still experiencing the issue?

Everything looks available....https://rubygems.org/gems/hitimes

Could it be a bundler issue? Perhaps try to update bundler itself?

**Comment by davidcpell** [comment]
Yeah, sorry about this. There's definitely something lower-level going on. I'm having this problem no matter what ActiveJob adapter I use. Delayed Job was getting the same error and then when I installed a suggested dependency ("daemons") I started getting the Passenger error about that one. Really no idea what's going on. I'll close this. 

Here's my Stack Overflow question if you don't mind taking a look at the discussion: http://stackoverflow.com/questions/28216969/rails-4-2-activejob-every-adapter-results-in-phusionpassenger-error-could-not
